src/main/resources/templates/pages/dashboard/index.peb

```diff
@@ -3,7 +3,7 @@
     {% extends "layouts/dashboard-layout" %}
     {% block title %} Home {% endblock %}
 {% block content %}
-<div data-test="dashboard-index"></div>
```

Comment:
dashboard-index can't be visible with a height of 0!

Reply:
But it can exist! Can you keep this as it was please? I'd like to be able to rely on CSS like `body > main > header` which breaks if we add a generation in between.

Reply:
We can have cypress assert that it exists